client/lib/checkout/validation.js

```diff
@@ -244,6 +244,13 @@ validators.validPhone = {
 	},
 };
 
+validators.validStreetNumber = {
+	isValid( streetNumber ) {
+		return streetNumber !== '0';
```

Comment:
It is true to consider `0` as an invalid street number (as far as I know), but we can add all negative numbers too.

Something like:

```
isValid( streetNumber ) {
    const parsedNumber = parseInt( streetNumber );

    return ( ! isNaN( parsedNumber ) ) && parsedNumber > 0;
}

Reply:
That would work, but I'm afraid it would be too restrictive. I'm not actually trying to validate the number itself. The field is supposed to be a string (think `22 bis`), but this patch is just meant to overcome what I think is a bug on Ebanx' side where 0 is interpreted as false (or empty). 

We've had some customers who added the street number to the street name, and then (because they were told that this field cannot be empty), added 0 as the street number. The transaction failed, while it would have succeeded with any other non-empty value on the street number.

Reply:
The case of `22 bis` is a valid counter-argument. Good!